src/main/webapp/feed.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Message Feed</title>
+<link rel="stylesheet" href="/css/main.css">
+<link rel="stylesheet" href="/css/user-page.css">
+<script src = "/js/feed-loader.js"></script>
+<script src="/js/navigation-loader.js"></script>
+</head>
+<body onload="buildUI()">
+  <nav>
```

Comment:
In the future, we should refactor this out, so that new pages don't have to copy paste it anymore.

Reply:
Noted